src/Features/CSharp/Portable/Completion/KeywordRecommenders/IntoKeywordRecommender.cs

```diff
@@ -40,6 +40,17 @@ private bool IsValidContextForSelect(CSharpSyntaxContext context)
 
             var lastToken = select.Expression.GetLastToken(includeSkipped: true);
 
+            var memberAccessExpression = select.Expression as MemberAccessExpressionSyntax;
+            if(memberAccessExpression  != null && string.IsNullOrEmpty(memberAccessExpression.Name.ToString()))
+            {
+                return false;
+            }
+
+            //if(lastToken.Kind() == SyntaxKind.DotToken)
```

Comment:
agreed.  the way most recommenders work is to get the previous complete token (you can look at many other recommenders to see how), and use that to proffer or eliminate options.  In this case, it seems rather trivial to just check if the previous complete token is a dot and not offer in that case.